Altis_Life.Altis/core/functions/fn_getIndex.sqf

```diff
@@ -0,0 +1,30 @@
+#include "..\..\script_macros.hpp"
+/*
+    File: fn_getIndex.sqf
+    Author: Bryan "Tonic" Boardwine
+    Edited by: Daniel Stuart (Moved to client-side and added support for simple arrays)
+
+    Description:
+    Return the index for the respective item in an array.
+
+    Parameter(s):
+        0: ANYTHING - Value to search for
+        1: ARRAY - array to be searched
+
+    Returns:
+    NUMBER
+*/
+params [
+    "_value",
+    ["_array",[],[[]]]
+];
+
+_return = -1;
+{
+    if ((_x isEqualType [] && {_value in _x}) || {_value isEqualTo _x}) then {
+        _return = _forEachIndex;
+        breakOut "";
+    };
+} forEach _array;
+
+_return;
```

Comment:
I'm 50/50 on this due to the fact the user could avoid using this (and use the other commands that do the same thing, internally). I know it's good to be 'well-rounded', but I think most people know when and when to not use this. It would seem silly to use this vs find. I like the idea of another function supporting a deeper search though (actual index path) on a side note. If we do stick with this, perhaps abstracting the logic out of the loop, so for large arrays, it doesn't need to run un-needed conditions every iteration?